app/views/organizations/page_texts/_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= bootstrap_form_with model: page_text, :url => page_text_path(@page_text), method: :patch do |form| %>
+  <div class='card-body'>
+    <div class="d-lg-flex align-items-center justify-content-between">
+      <h3 class="mb-1">Landing page details</h3>
+      <p class="text-muted mb-3" style="font-size: 12px;">Edit your landing page.
+      <p>
+    </div>
+    <div class="row mt-3">
+      <div class="col-lg-6">
+        <%= form.fields_for :page_text do |page_text_form| %>
+          <form class="mb-3">
+            <!-- Hero Text -->
+            <div class="form-group">
+              <%= form.text_field :hero,
+                                  label: "Hero",
+                                  autofocus: true,
+                                  placeholder: "hero text",
+                                  class: 'form-control' %>
+            </div>
+            <!-- About Us -->
+            <div class="form-group">
+              <%= form.text_field :about, label: "About Us", placeholder: "About Us text", class: 'form-control' %>
```

Comment:
Can we change this to a text area instead? This input should easily accept a couple hundred words so it would be nice to see that easier in a text area.